roles/kubernetes/preinstall/handlers/main.yml

```diff
@@ -78,6 +78,9 @@
 
 - name: Preinstall | restart kube-controller-manager crio/containerd
   shell: "{{ bin_dir }}/crictl pods --name kube-controller-manager* -q | xargs -I% --no-run-if-empty bash -c '{{ bin_dir }}/crictl stopp % && {{ bin_dir }}/crictl rmp %'"
+  register: preinstall_restart_controller_manager
+  retries: 10
+  until: preinstall_restart_controller_manager.rc == 0
```

Comment:
Why don't we specify `delay: 1`?
apiserver reries contains `delay: 1`. The default value is 5 if not specifying.

In addition, did you face a situation we need to add retries here?
If so, please write some error message in the pull request description if possbile.

Reply:
I add `delay: 1` to this pr.

And i add the error description.